src/pages/Login.tsx

```diff
@@ -55,35 +48,67 @@ const Login = () => {
                     Your JWT Token expired. Please login again.
                 </Alert>
             </Snackbar>
-            <Topbar isNavigationOpen={false} />
-            <Grid item xs={3}>
-                <Card elevation={24} sx={{ maxWidth: 400, minHeight: 300 }}>
-                    <CardContent>
+
+            <Paper
+                sx={{
+                    maxHeight: 800,
+                    display: 'flex',
+                    flexFlow: 'row wrap',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    p: 3,
+                    background:
+                        'linear-gradient(179.6deg, rgba(99, 138, 169, 0.24) 0%, rgba(13, 43, 67, 0.2) 76.56%, rgba(13, 43, 67, 0.1) 100%)',
```

Comment:
Don't know why I didn't think about this until now... How are we deciding on gradients ?